controllers/core/configmap_controller_test.go

```diff
@@ -127,7 +141,7 @@ func Test_Reconcile_ConfigMap_PD_Disabled_If_IPAM_Disabled(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 
-	mock := NewConfigMapMock(ctrl, mockConfigMapPD)
```

Comment:
> Seems mockConfigMapPD is no longer used in this file, let's delete it if not used.

Correct. I removed it

Reply:
> On a side note, we are missing tests cases for when IPAM is enabled, PD is disabled, but IPAM config is updated. Can we add this test case?

Good catch, thanks for that suggestion. I have since added a test for the scenario you descrived.